src/Magnum/Platform/WindowlessEglApplication.cpp

```diff
@@ -476,7 +526,7 @@ bool WindowlessEglApplication::tryCreateContext(const Configuration& configurati
     Configuration mergedConfiguration{configuration};
     #ifndef MAGNUM_TARGET_WEBGL
     if(!mergedConfiguration.device())
-        mergedConfiguration.setDevice(_commandLineDevice);
+        mergedConfiguration.setDevice(_commandLineDevice).setDeviceIsCuda(_commandLineDeviceIsCuda);
```

Comment:
```suggestion
        mergedConfiguration.setDevice(_commandLineDevice).setCudaDevice(_commandLineCudaDevice);
```

ok sorry at this point i'm just using you as a remote keyboard :sweat_smile: